templates/base.html

```diff
@@ -21,9 +21,21 @@
         <meta name="author" content="{{ AUTHOR }}" />
         {% endif %}
 
+        {% if article and article.featured_image %}
```

Comment:
I suggest moving out this patch from base.html to smo_metadata file and then include it in the base.html

Reply:
How should I include it on base so that it works with all elements (article, page, landing etc ? )